src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.wasm.cs

```diff
@@ -99,7 +99,16 @@ void OnJsonChanged(string updatedJson, string updatedCacheKey)
 				}
 				else
 				{
-					var documentPath = Windows.Storage.Helpers.AssetsPathBuilder.BuildAssetUri(UriSource?.PathAndQuery);
+					var documentPath = string.Empty;
+				
+					if(UriSource.Scheme.Equals("http") || UriSource.Scheme.Equals("https"))
+					{
+						documentPath = UriSource.ToString();
```

Comment:
`ToString()` generally removes escaped characters, and that can be a problem on some servers:
```suggestion
						documentPath = UriSource.OriginalString;
```